website/docs/docs/deploy/deployment-tools.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Deploy with other tools"
+id: "deployment-tools"
+sidebar: "Deploy with other tools"
+---
+
+Discover effortless ways to schedule and run your dbt jobs with the help of powerful tools like [dbt Cloud](/docs/deploy/dbt-cloud-job), Airflow, Prefect, Dagster, automation server, Cron, or Azure Data Factory (ADF). 
+
+Use these tools to automate your data workflows, trigger dbt jobs (including those hosted on dbt Cloud), and enjoy a hassle-free experience, saving time and increasing efficiency.
+
+
+## Airflow
+
+If your organization is using [Airflow](https://airflow.apache.org/), there are a number of ways you can run your dbt jobs, including:
+
+<Tabs>
+
+<TabItem value="airflowcloud" label="dbt Cloud">
+
+Installing the [dbt Cloud Provider](https://registry.astronomer.io/providers/dbt-cloud) to orchestrate dbt Cloud jobs. This package contains multiple Hooks, Operators, and Sensors to complete various actions within dbt Cloud.
+
+<Lightbox src="/img/docs/running-a-dbt-project/airflow_dbt_connector.png" title="Airflow DAG using DbtCloudRunJobOperator"/>
+<Lightbox src="/img/docs/running-a-dbt-project/dbt_cloud_airflow_trigger.png" title="dbt Cloud job triggered by Airflow"/>
+
+</TabItem>
+
+<TabItem value="airflowcore" label="dbt Core">
+
+Invoking dbt Core jobs through the [BashOperator](https://registry.astronomer.io/providers/apache-airflow/modules/bashoperator). In this case, be sure to install dbt into a virtual environment to avoid issues with conflicting dependencies between Airflow and dbt.
+
+</TabItem>
+</Tabs>
+
+For more details on both of these methods, including example implementations, check out [this guide](https://www.astronomer.io/guides/airflow-dbt).
+
+## Azure Data Factory
+
+Integrate dbt Cloud and [Azure Data Factory](https://learn.microsoft.com/en-us/azure/data-factory/) (ADF) for a smooth data process, from data ingestion to data transformation. You can seamlessly trigger dbt Cloud jobs upon completion of ingestion jobs by using the [dbt API](/docs/dbt-cloud-apis/overview) in ADF. 
+
+The following video provides you with a detailed overview of how to trigger a dbt Cloud job via the API in ADF. 
+
+Need help building this out? - [Contact us](https://www.getdbt.com/contact/) today!
```

Comment:
I think this stray sentence might be better at the end of the 1st paragraph in line 38